compilers/concrete-compiler/compiler/CMakeLists.txt

```diff
@@ -149,6 +149,26 @@ if(CONCRETELANG_CUDA_SUPPORT)
   add_compile_options(-DCONCRETELANG_CUDA_SUPPORT)
 endif()
 
+# --------------------------------------------------------------------------------
```

Comment:
Add a cmake variable to disable on macos and on Makefile this variable is OFF by default on macos and ON on other platform.